src/plugins/data/public/search/search_interceptor/search_interceptor.ts

```diff
@@ -535,6 +536,7 @@ export class SearchInterceptor {
     } else {
       this.deps.toasts.addError(e, {
         title: 'Search Error',
+        fingerprintId: String(sha256.create().update(JSON.stringify(e))),
```

Comment:
it seems `sha256` is adding +10.7KB to the plugin, there shall be a cheaper version, it could also be just the stringifyed Error message. What this code does is preventing duplicate errors